homeassistant/components/fritz/sensor.py

```diff
@@ -66,6 +66,11 @@ def _retrieve_external_ip_state(status: FritzStatus, last_value: str) -> str:
     return status.external_ip  # type: ignore[no-any-return]
 
 
+def _retrieve_external_ipv6_state(status: FritzStatus, last_value: str) -> str:
+    """Return external ipv6 from device."""
+    return status.external_ipv6  # type: ignore[no-any-return]
```

Comment:
```suggestion
    return cast(str, status.external_ipv6)
```

If this is always a string

Reply:
You may need to import `cast from typing`

Reply:
All sensors in the file use the type ignore hint. Just tried to keep it consistent. 
Will change it when I find my way to a PC.